thread_safe_function_round_trip/node-api/round_trip.c

```diff
@@ -352,14 +356,16 @@ static void addon_is_unloading(napi_env env, void* data, void* hint) {
   assert(uv_mutex_init(&(addon_data->check_status_mutex)) == 0);
 
   napi_value thread_item_class;
-
+  napi_property_descriptor thread_item_properties[] = {
+    { "prime", 0, 0, GetPrime, 0, 0, napi_default, 0 }
```

Comment:
@amissine `napi_enumerable` might be better than `napi_default` because then the property can be seen by, for example, `JSON.stringify()`.

Reply:
Thanks! Fixing...